fastlane/lib/fastlane/actions/nexus_upload.rb

```diff
@@ -32,6 +32,11 @@ def self.upload_options(params)
         options << "-F g=#{params[:repo_group_id].shellescape}"
         options << "-F a=#{params[:repo_project_name].shellescape}"
         options << "-F v=#{params[:repo_project_version].shellescape}"
+
+        if params.fetch(:repo_classifier, ask: false)
```

Comment:
Thanks, I just learned Ruby to do this so forgive my non-idiomaticness. I just pushed the change.